src/TypeSpec.Extension/Emitter.Csharp/test/Unit/propertyType.test.ts

```diff
@@ -198,6 +199,7 @@ describe("Test GetInputType for enum", () => {
                         }
                     ],
                     IsExtensible: false,
+                    IsLowConfident: false,
```

Comment:
From the PR description, there are multiple combination of LowConfident cases, could you add more testcases to ensure more scenarios are covered?

Reply:
will add more test cases in the unit test of the emitter
not covered right now

Reply:
unit tests added